x-pack/elastic-agent/spec/apm-server.yml

```diff
@@ -1,7 +1,12 @@
 name: APM-Server
 cmd: apm-server
 artifact: apm-server
-args: ["-E", "management.enabled=true", "-E", "management.mode=x-pack-fleet", "-E", "apm-server.data_streams.enabled=true"]
+args: [
+  "-E", "management.enabled=true",
+  "-E", "management.mode=x-pack-fleet",
+  "-E", "apm-server.data_streams.enabled=true"
+  "-E", "gc_percent=${APMSERVER_GOGC:100}"
```

Comment:
This works because libbeat merges all -E flags into the configuration. The environment variable with default will be evaluated by libbeat.